### PR TITLE
Add optional yield to run

### PR DESCRIPTION
**Pull request by bhenderson**
My main issue was needing to support sudo AND a custom password prompt.

### CODE REVIEW

**Comment by bhenderson** [comment]
what do you think?

File: Rakefile [hunk 1]
```diff
@@ -14,6 +14,8 @@ Hoe.spec 'rake-remote_task' do
   dependency 'rake',    ['>= 0.8', '< 11.0']
   dependency 'open4',    '~> 1.0'
 
+  dependency 'minitest', '~> 5.0', :dev
+
```
Comment (zenspider) [inline Rakefile]: this is done automatically by my hoe setup.

**Comment by zenspider** [comment]
Seems fine. Is this testable in some way?

**Comment by bhenderson** [comment]
how's this?

also, what do you do in your hoe setup to get minitest not to get nuked?

**Comment by zenspider** [comment]
Just having minitest & hoe installed should be enough.

**Comment by zenspider** [comment]
Done. Thanks!

**Pull request closed without merging**